book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -1,16 +1,16 @@
 [[_advanced_merging]]
-=== Advanced Merging
+=== Advanced na Merging
 
-Merging in Git is typically fairly easy.
-Since Git makes it easy to merge another branch multiple times, it means that you can have a very long lived branch but you can keep it up to date as you go, solving small conflicts often, rather than be surprised by one enormous conflict at the end of the series.
+Ang merging sa Git ay karaniwang medyo madali.
+Dahil ang Git ay nagagawa nito ipadali ang pag-merge sa ibang branch ng maraming beses, ito ay nangangahulugan na maaari ka na merong isang matagal nang buhay na branch ngunit maaari mong panatilihin ang pinakabago habang ikaw ay gumagamit, paglutas ng maliit na mga kasalungat ng madalas, sa halip na mabigla ng isang napakalaking salungat sa katapusan ng serye.
 
-However, sometimes tricky conflicts do occur.
-Unlike some other version control systems, Git does not try to be overly clever about merge conflict resolution.
-Git's philosophy is to be smart about determining when a merge resolution is unambiguous, but if there is a conflict, it does not try to be clever about automatically resolving it.
-Therefore, if you wait too long to merge two branches that diverge quickly, you can run into some issues.
+Gayunpaman, minsan nakakalito ang mga kasalungat na magaganap.
+Di tulad ng ibang mga bersyon na kontrol na mga sistema, ang Git ay hindi susubok upang maging sobrang matalino tungkol sa pagsama ng pagsasalungatan na resolusyon.
+Ang pilisopiya ng Git ay dapat maging talino tungkol sa pagtukoy kapag ang isang merge na resolusyon ay hindi malabo, ngunit kung may isang salungat, hindi ito sinubukan na maging matalino tungkol sa awtomatikong palutas nito.
+Samakatuwid, kung ikaw ay maghintay ng sobrang matagal upang i-merge ang dalawang branch na mabilis na nagbabagu-bago, maaari kang magpatakbo ng ilang mga isyu.
```

Comment:
``maghintay ng sobrang matagal``
to
``naghintay nang sobrang tagal``

Reply:
done change